ccd-definitions/src/test/java/uk/gov/hmcts/reform/divorce/ccd/framework/StubConfigBuilder.java

```diff
@@ -0,0 +1,264 @@
+package uk.gov.hmcts.reform.divorce.ccd.framework;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.gov.hmcts.ccd.sdk.types.ConfigBuilder;
+import uk.gov.hmcts.ccd.sdk.types.Event;
+import uk.gov.hmcts.ccd.sdk.types.EventTypeBuilder;
+import uk.gov.hmcts.ccd.sdk.types.Field;
+import uk.gov.hmcts.ccd.sdk.types.HasRole;
+import uk.gov.hmcts.ccd.sdk.types.RoleBuilder;
+import uk.gov.hmcts.ccd.sdk.types.Search;
+import uk.gov.hmcts.ccd.sdk.types.Tab.TabBuilder;
+import uk.gov.hmcts.ccd.sdk.types.WebhookConvention;
+import uk.gov.hmcts.ccd.sdk.types.WorkBasket;
+import uk.gov.hmcts.reform.divorce.ccd.model.CaseData;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Hashtable;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+@Getter
+@SuppressWarnings("rawtypes")
+public class StubConfigBuilder<T, S, R extends HasRole> implements ConfigBuilder<T, S, R> {
```

Comment:
You might have to run me through this, I thought we'd just create a mock with mockito?

Reply:
I think we will need to discuss the testing strategy on what we want to test and how.

Reply:
Mocking the builder proved difficult and messy, the idea here was to create the stubs and a set of matchers (not implemented yet), so that input into the builders can be easily checked.